docs/.gitbook.yaml

```diff
@@ -0,0 +1 @@
+root: ./docs/
```

Comment:
@ascerra Is the plan to continue using GitBook or are we looking to use [GitHub pages](https://pages.github.com)? Since these docs are hosted on GitHub, may be worth looking into using pages. Which then could allow for using GitHub Action for auto deploying changes.

Reply:
This is a good suggestion, I'll look into getting this done with github pages, thanks for the comment

Reply:
@ascerra I primarily presented the question about using GitHub pages as it would mean its a one stop shop for everything (e.g. GitHub = repositories, docs).

I am totally okay to keep with gitbook for building/deploying documentation if that is easier to work with over using Jekyll. Please feel free to stop on this path and continue on with gitbook, if that works best. :)

Reply:
Excellent thanks for the feedback Ryan!

We'll still be using this github repo as the source of truth for the documentation with gitbook synchronizing each time new code is merged into main.

Reply:
Sounds great @ascerra!